Source/WebCore/dom/Document.cpp

```diff
@@ -1327,6 +1329,22 @@ MediaQueryMatcher& Document::mediaQueryMatcher()
     return *m_mediaQueryMatcher;
 }
 
+const RenderStyle* Document::mediaQueryDefaultStyle() const
+{
+    if (m_cachedMediaQueryDefaultStyle == nullptr) {
+        auto defaultStyle = RenderStyle::createPtr();
```

Comment:
Another WebKit coding style issue. This isn’t my personal preference, but it’s what we’re doing as a project. Recently, to make lifetime clearer, we have been writing out smart pointer types rather than using `auto`. No need to be explicit about the type of the pointed-to object, but we are explicit about the type of pointer. So we write `Ref` or `RefPtr` or whatever.
```suggestion
        std::unique_ptr defaultStyle = RenderStyle::createPtr();
```